src/style.css

```diff
@@ -1 +1,93 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+
+  --main-color: #00acdc;
+}
+
+li {
+  list-style: none;
+
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+.container {
+  padding: 0 50px;
+  margin: 0 auto;
+  max-width: 1200px;
```

Comment:
There is no need to limit the maximum width of the header, you can make it responsive

Reply:
updated all